core/math/basis.cpp

```diff
@@ -773,7 +773,7 @@ Basis::operator String() const {
 				mtx += ", ";
 			}
 
-			mtx += rtos(elements[i][j]);
+			mtx += rtos(elements[j][i]); //matrix is stored transposed for performance, so print it transposed
```

Comment:
@reduz This change doesn't seem to be related to SDFGI, was there a reason why it was changed in this PR?

Regardless, #34668 makes Basis and Transform(2D) print 2-dimensionally, which avoids confusion.